topics/sequence-analysis/tutorials/genetic-map-rad-seq/tutorial.md

```diff
@@ -24,7 +24,7 @@ We here proposed to re-analyze these data at least until genotypes determination
 
 ## Data upload
 
-The original data is available at [STACKS website](https://creskolab.uoregon.edu/stacks/tutorial/stacks_samples.tar.gz) and the subset used here is findable on [CeSGO hub](https://cesgo.genouest.org/resources/370/supportingdocs).
+The original data is available at [STACKS website](http://catchenlab.life.illinois.edu/stacks/param_tut.php) and the subset used here is findable on [CeSGO hub](https://cesgo.genouest.org/resources/370/supportingdocs).
```

Comment:
I can't find any data in either websites (the second just redirects to https://www.cesgo.org/en/ )

Reply:
ping @yvanlebras